web/src/components/CategoryListItem.tsx

```diff
@@ -65,25 +42,39 @@ type CategoryListItemProps = {
 }
 
 const CategoryListItem = ({ category, subCategories }: CategoryListItemProps): ReactElement => {
-  const SubCategories = subCategories.map(subCategory => (
-    <SubCategory key={subCategory.path} dir='auto'>
-      <StyledLink to={subCategory.path}>
-        {!!subCategory.thumbnail && <CategoryThumbnail alt='' src={subCategory.thumbnail} />}
-        <SubCategoryCaption>{subCategory.title}</SubCategoryCaption>
-      </StyledLink>
-      <Divider />
-    </SubCategory>
-  ))
+  const SubCategories = subCategories.map(subCategory => {
+    const { path, thumbnail, title } = subCategory
+    return (
+      <SubCategoryListItem key={path}>
+        <StyledSubCategoryListItemButton component={Link} to={path}>
+          {!!thumbnail && (
+            <ListItemAvatar>
+              <Avatar src={thumbnail} variant='square' />
+            </ListItemAvatar>
+          )}
+          <ListItemText primary={<Typography variant='body1'>{title}</Typography>} />
+        </StyledSubCategoryListItemButton>
+      </SubCategoryListItem>
+    )
+  })
 
   return (
-    <Row>
-      <StyledLink dir='auto' to={category.path}>
-        {!!category.thumbnail && <CategoryThumbnail alt='' src={category.thumbnail} />}
-        <CategoryItemCaption>{category.title}</CategoryItemCaption>
-      </StyledLink>
-      <Divider />
-      <SubCategoriesContainer>{SubCategories}</SubCategoriesContainer>
-    </Row>
+    <StyledListItem disablePadding>
+      <ListItemButton component={Link} to={category.path} dir='auto'>
```

Comment:
❓ Is this direction still necessary?
```suggestion
      <ListItemButton component={Link} to={category.path}>
```

Reply:
Nice catch ... no need for it. and fixes the icon alignment for RTL 👍️